ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/API.hs

```diff
@@ -164,16 +165,19 @@ data BlockInfo blk = BlockInfo {
 ------------------------------------------------------------------------------}
 
 -- | Errors which might arise when working with this database.
-data VolatileDBError =
-    -- | An error thrown because of incorrect usage of the VolatileDB
-    -- by the user.
+data VolatileDBError blk =
+
     ApiMisuse ApiMisuse
+    -- ^ An error thrown because of incorrect usage of the VolatileDB
+    -- by the user.
+
```

Comment:
We prefer the previous style (comment before constructor) for sum types. (Also, mixing two styles in data type is rather inconsistent).